pkg/monitoring/metrics/virt-handler/domainstats/vcpu_metrics.go

```diff
@@ -75,7 +75,7 @@ func (vcpuMetrics) Collect(vmiReport *VirtualMachineInstanceReport) []operatorme
 				"id":    stringVcpuIdx,
 				"state": humanReadableState(vcpu.State),
 			}
-			crs = append(crs, vmiReport.newCollectorResultWithLabels(vcpuSeconds, microsecondsToSeconds(vcpu.Time), additionalLabels))
+			crs = append(crs, vmiReport.newCollectorResultWithLabels(vcpuSeconds, nanosecondsToSeconds(vcpu.Time), additionalLabels))
```

Comment:
Looks like [`microsecondsToSeconds`](https://github.com/brandboat/kubevirt/blob/f96d08b011eae2af4628c93bc596bead8683f5b9/pkg/monitoring/metrics/virt-handler/domainstats/unit_converter.go#L7) is not used anymore.
Can you drop it? Thank you!

Reply:
Sure! I'll remove it. Thanks for the comment!